lib/util/exec/docker/index.ts

```diff
@@ -84,7 +84,11 @@ async function getDockerTag(
     { constraint },
     `Found ${scheme} version constraint - checking for a compatible ${depName} image to use`
   );
-  const imageReleases = await getPkgReleases({ datasource: 'docker', depName });
+  const imageReleases = await getPkgReleases({
+    datasource: 'docker',
+    depName,
+    versioning: scheme,
```

Comment:
I'm not sure if this is the right scheme we want to use. 🤔 But if i check the code, only `isVersion` and `sortVersions` are used.

Reply:
Why wouldn't it be? Wasn't the purpose of passing the scheme because the constraints could be in language-specific syntax? e.g. pep440 which wouldn't be compatible with semver